src/material/core/theming/tests/test-theming-api.scss

```diff
@@ -202,6 +205,28 @@ $dark-theme-only-typography: mat-dark-theme((typography: $typography-config));
   @include _my-custom-theme-new-api($new-api-dark-theme);
 }
 
+// Test which ensures that either theme objects, or individual system configurations
+// can be passed to the all-density, all-typography or all-color mixins.
+@mixin test-all-theme-mixins-arguments() {
+  $test-themes: (
+    $legacy-light-theme,
+    $legacy-dark-theme,
+    $new-api-light-theme,
+    $new-api-dark-theme
+  );
+
+  @each $theme in $test-themes {
+    @include angular-material-typography($theme);
+    @include angular-material-color($theme);
+    @include _angular-material-density($theme);
+    @include _angular-material-density($theme);
```

Comment:
Seems like this call is redundant, because it's the same as the one above.

Reply:
Fixed!